File: app/models/comment.rb
class Comment < ApplicationRecord
  belongs_to :ticket
  belongs_to :made_by, class_name: "User"

  validates :made_by, :ticket, :content, presence: :true
  validates :content, length: { maximum: 1000 }

  attr_readonly :made_by_id, :ticket_id
end

File: app/javascript/components/CommentsSection/Comment.js
import React from "react";

function Comment({ madeBy, content, createdAt }) {
  return (
    <tr>
      <td className="fs-4">{madeBy}</td>
      <td className="fs-4">{content}</td>
      <td className="fs-4">{createdAt}</td>
    </tr>
  );
}

export default Comment;
